rotkehlchen/tests/unit/test_data_updates.py

```diff
@@ -553,3 +603,52 @@ def test_accounting_rules_updates(data_updater: RotkiDataUpdater) -> None:
             'counterparty': 'test_counterparty',
         },
     ]
+
+
+def _check_location_asset_mappings(cursor: 'DBCursor', after_upgrade: bool) -> None:
+    """Auxiliar function to check the db values before and after the upgrade"""
```

Comment:
This is not Spanish :sweat_smile: 

```suggestion
    """Auxiliary function to check the db values before and after the upgrade"""
```